django/forms/fields.py

```diff
@@ -762,14 +763,19 @@ class URLField(CharField):
 
     def __init__(self, *, assume_scheme=None, **kwargs):
         if assume_scheme is None:
-            warnings.warn(
-                "The default scheme will be changed from 'http' to 'https' in Django "
-                "6.0. Pass the forms.URLField.assume_scheme argument to silence this "
-                "warning.",
-                RemovedInDjango60Warning,
-                stacklevel=2,
-            )
-            assume_scheme = "http"
+            if settings.EARLY_ADOPT_URLFIELD_ASSUME_HTTPS is True:
```

Comment:
Allow truthy values?

```suggestion
            if settings.EARLY_ADOPT_URLFIELD_ASSUME_HTTPS:
```